README.md

```diff
@@ -16,6 +16,11 @@ The purpose of this docker image is to help in the development of ceph.
 ### Running the container
 
     # docker run -it -v /home/rimarques-local/projects/ceph:/ceph --net=host ceph-dev-docker /bin/bash
+
+<aside class="warning">
```

Comment:
Enclosing the text block with these tags seems to disable Markup rendering - at least the output generated by clicking "View" looks strange.